libs/ui/src/components/NeoButton/NeoButton.vue

```diff
@@ -24,20 +25,26 @@
 import { OButton } from '@oruga-ui/oruga'
 import { NeoButtonVariant } from '@kodadot1/brick'
 
-defineProps<{
-  size?: 'small' | 'medium' | 'large'
-  disabled?: boolean
-  expanded?: boolean
-  icon?: string
-  iconPack?: string
-  label?: string
-  active?: boolean
-  fixedWidth?: boolean
-  noShadow?: boolean
-  variant?: NeoButtonVariant
-  rounded?: boolean
-  tag?: string
-}>()
+withDefaults(
```

Comment:
Don't need `withDefaults`
Declaring  optional prop  `loadingWithLabel? : boolean`

Also works


other than that lgtm